waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/server/FederatedHMSHandlerTest.java

```diff
@@ -156,15 +214,15 @@ public void get_database() throws Exception {
   }
 
   @Test
-  public void drop_database() throws NoSuchObjectException, InvalidOperationException, MetaException, TException {
+  public void drop_database() throws TException {
```

Comment:
For the tests I don't really care whether we call out all the different exceptions or just the base `TException`, it's less verbose and less to read this way so fine.

Reply:
+1